app/helpers/patient_query_helper.rb

```diff
@@ -175,7 +175,11 @@ def filter_by_text(patients, search)
         patients.where('lower(user_defined_id_statelocal) like ?', "#{search&.downcase}%").or(
           patients.where('lower(user_defined_id_cdc) like ?', "#{search&.downcase}%").or(
             patients.where('lower(user_defined_id_nndss) like ?', "#{search&.downcase}%").or(
-              patients.where('date_of_birth like ?', "#{search&.downcase}%")
+              patients.where('date_of_birth like ?', "#{search&.downcase}%").or(
+                patients.where('lower(patients.email) like ?', "#{search&.downcase}%").or(
```

Comment:
so I know the other parts of this query were already like this before, but I think we actually don't need to use the `lower` method in the SQL query because our database performs case insensitive searches on strings already and by using the `lower` method on these, we actually make the query much less performant because the database can no longer use the index that we have on these fields and has to do a full table scan instead:

<img width="1260" alt="Screen Shot 2021-11-26 at 2 06 55 PM" src="https://user-images.githubusercontent.com/9956561/143622272-455db14f-beb7-49ce-8ce3-dc95e9440f16.png">
<img width="1200" alt="Screen Shot 2021-11-26 at 2 07 02 PM" src="https://user-images.githubusercontent.com/9956561/143622273-9ba16f32-baf4-4042-af02-a9e5aeff9664.png">

Reply:
@timwongj do you want me to remove all the `lower` uses in this method then?

Reply:
yeah, can you also do that for the `close_contact_query_helper`?

Reply:
yup, done: https://github.com/SaraAlert/SaraAlert/pull/1164/commits/735d0d794dccc9773fcc41cddffc0619a4fa5e03